### PR TITLE
🎙️ Podcast Framework

### PR DESCRIPTION
**Pull request by itsmingjie**

### CODE REVIEW

**Comment by brndnwll** [comment]
ur so hot for what reason

File: layouts/podcast/single.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  }
+</style>
+{{ end }}
+{{ define "main" }}
+<div class="single-podcast">
+  <section class="hero is-small is-dark">
+    <div class="hero-body">
+      <div class="container post-header">
+        <h1 class="title is-3 mb-2">
+          Ep.{{.Params.episode}}: {{.Title}}
```
Comment (brndnwll) [inline layouts/podcast/single.html]: Would do `EP{{.Params.episode}}: {{.Title}}` instead? Kinda nitpicky but not a big fan of the "Ep. 1 -"

File: layouts/podcast/summary.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="summary">
+  <div class="titles">
+    <a href="{{ .RelPermalink }}">
+      <h2 class="title">Ep.{{.Params.episode}}: {{ .Title }}</h2>
```
Comment (brndnwll) [inline layouts/podcast/summary.html]: Same as above: `EP{{.Params.episode}}: {{.Title}}`

File: layouts/podcast/single.html (hunk 1 above)
Comment (itsmingjie) [reply layouts/podcast/single.html]: Fixed in de62bf3!

File: layouts/podcast/summary.html (hunk 2 above)
Comment (itsmingjie) [reply layouts/podcast/summary.html]: Fixed in de62bf3!

**Review by brndnwll — COMMENTED** [review]
Looks so gorg!! Wdyt of links over web bookmarks?

**Pull request merged**